### ISSUE TITLE
FXIOS-88 ⁃ [dynamic fonts] Firefox addressbar doesn't respect Larger Text accessibility setting

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/firefox-ios#5033]
If you set iOS to use Larger Text, Firefox's addressbar text does not increase in size. It probably should.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-88)

**Comment by SimonBasca** [comment]
@dnarcese it seems that after 0467e84007b landed there was a regression introduced, #7837. The introduced issues are affecting user experience and should be fixed before #5033 lands on beta.

**Comment by SimonBasca** [comment]
Verified fix on 7e54b8b95.
